legend-engine-ide-lsp-default-extensions/src/main/java/org/finos/legend/engine/ide/lsp/extension/mapping/MappingLSPGrammarExtension.java

```diff
@@ -188,7 +189,7 @@ public Iterable<? extends LegendExecutionResult> execute(SectionState section, S
             }
             default:
             {
-                return super.execute(section, entityPath, commandId, executableArgs, Map.of());
+                return FunctionExecutionSupport.execute(this, section, entityPath, commandId, executableArgs, inputParams);
```

Comment:
We dont need this on mapping extension.

Reply:
Makes sense. Reverted the change to this file.